versions/QmctngrQAt9fjpQUZr7Bx3BsXUcif52eZGTizWhvcShsjz/static/js/Detector.js

```diff
@@ -5,56 +5,50 @@
 
 Detector = {
 
-  canvas : !! window.CanvasRenderingContext2D,
```

Comment:
The versions should be changed after release, `versions` should be included from linting etc